cartographer/mapping_2d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -184,14 +225,16 @@ void ConstraintBuilder::ComputeConstraint(
   const sensor::PointCloud filtered_point_cloud =
       adaptive_voxel_filter_.Filter(*point_cloud);
 
-  // The 'constraint_transform' (i <- j) is computed from:
-  // - a 'filtered_point_cloud' in j,
-  // - the initial guess 'initial_pose' for (map <- j),
-  // - the result 'pose_estimate' of Match() (map <- j).
-  // - the ComputeSubmapPose() (map <- i)
+  // The 'constraint_transform' (submap <- scan) is computed from:
+  // - a 'filtered_point_cloud' in scan,
+  // - the initial guess 'initial_pose' for (map <- scan),
+  // - the result 'pose_estimate' of Match() (map <- scan).
+  // - the ComputeSubmapPose() (map <- submap)
   float score = 0.;
   transform::Rigid2d pose_estimate = transform::Rigid2d::Identity();
 
+  // Compute pose_estimate in three stages: Fast estimate, pruning, refinement
+  // First estimate using fast correlative scan matcher; return if score is too low.
```

Comment:
nit: `;` -> `and`
Also: Did you run clang-format? Lines should be up to 80 characters usually.